data/campaigns/Northern_Rebirth/scenarios/13a_Showdown.cfg

```diff
@@ -844,11 +844,11 @@
                 [/message]
                 [message]
                     id=Eryssa
-                    message= _ "Good luck, you monster. You are cut off and surrounded. We shall kill you and feed your carcass to the vultures and wild beasts."
+                    message= _ "Good luck, you monster. You are cut off and surrounded. We shall kill you and feed your carcass to the vultures!"
                 [/message]
                 [message]
                     id=Rakshas
-                    message= _ "Hahahahah! You fools, do you not see that you are right in the heart of orcish territory, surrounded yourselves! Why, at this very moment my entire western army is marching this way. They will soon be here and will scatter your forces like straw!"
+                    message= _ "Hahahahah! You fools, do you not see that you are in the heart of orcish territory? You've surrounded yourselves! Why, at this very moment my entire western army is marching this way. They will soon be here and will scatter your forces like chaff!"
```

Comment:
Rakshas' line here is in response to Eryssa's claim that they have surrounded him. 'You've surrounded yourselves' doesn't really make sense to me.

Reply:
I like the new way. Anyone else want to weigh in?

Reply:
It made sense to me too? Like a snide, oh you think you're surrounding me? No, you're surrounding yourselfs hahahaha!! It's kind of anime-like but NR is kind of anime-like so it seemed okay to me.

Reply:
Eh, I wondered if that was the intention, but it still sounded odd. Whatever, if it works for you, fine.

Reply:
Needs a typographic apostrophe. No opinion on the change itself.

Reply:
Did it.